### DISCUSSION TITLE
Scroll

### DISCUSSION THREAD

**Discussion by EliabHdez** [Eugeny/tabby#3462] [General]
How I can hidden or disable the scrollbar??? And how I can scroll without use the mouse, that I can do it with the keyboard??? My apologize for my english, but I dont speak the language... Regards

**Comment by Eugeny** [comment]
You can hide scrollbars globally via this CSS snippet (add it to Settings -> Application -> Custom CSS): `::-webkit-scrollbar { display: none; }`

**Reply by EliabHdez** [reply]
Perfect!!! Thanks so much. Now there is some way to resize the panes with the keyboard??? And finally  I found a problem and this is that use the comand "exit" close the tab but does not close the terminal,  close the terminal manually and when I open it again, it opens with a tab with the title "exit" and I cant work in this tab!!!

**Reply by clarkwang** [reply]
I know almost nothing about web tech. Where can I find the doc for all these settings?

**Reply by Eugeny** [reply]
@EliabHdez there's no way to resize panes with keyboard shortcuts

@clarkwang it's just googling (e.g.: [::-webkit-scrollbar](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar) and trying things out. You can use the built-in dev tools (Settings -> Applications -> Devtools) to try out style changes while seeing them live.

**Reply by clarkwang** [reply]
What about `tab-header .index` (mentioned in #3464)? Is it also a "standard" property for browsers?

**Reply by Eugeny** [reply]
No - these are elements that constitute Terminus' interface. You can check out this guide on how to get started with browsing the element tree and changing properties: https://developers.google.com/web/tools/chrome-devtools/css

**Reply by clarkwang** [reply]
Cool. Managed to change the tab index color:

![image](https://user-images.githubusercontent.com/157076/108851780-c40a3880-761f-11eb-8292-8b37c12a930a.png)

**Comment by snailwei** [comment]
It'll be good to have a toggle option to turn off the scrollbar, it takes the terminal spaces. Thanks!
